go.mod

```diff
@@ -94,7 +94,7 @@ require (
 	github.com/prometheus/client_golang v1.9.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.15.0
-	github.com/rancher/aks-operator v1.0.1-rc4
+	github.com/rancher/aks-operator v1.0.1-rc6
```

Comment:
Whenever it changes here it also needs to be udpated here:https://github.com/rancher/rancher/blob/master/pkg/apis/go.mod#L9

Reply:
Done